package.json

```diff
@@ -3182,7 +3182,7 @@
                 "command": "aws.ecs.openTaskInTerminal",
                 "title": "%AWS.ecs.openTaskInTerminal%",
                 "category": "%AWS.title%",
-                "enablement": "viewItem == awsEcsContainerNodeExecEnabled && !isWeb",
```

Comment:
This and 2 other cases have non-trivial conditions and cloud9 doesn't support parentheses in the `when` (or `enablement` ?) clause. But these items wouldn't show in a web context anyway (because these explorer nodes won't show), so I just removed the `isWeb` check.